package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ndustrial/contxt-sdk",
-  "version": "0.0.23",
+  "version": "0.0.24",
```

Comment:
This should have happened after a CHANGELOG update

Reply:
dammit. I knew I was forgetting something.